### PR TITLE
First chart commit

### PR DESCRIPTION
**Pull request by T-Kukawka**
### Testing

Description on how akv2k8s can be tested.

- [ ] fresh install works
  - [ ] AWS
  - [x] Azure
  - [ ] KVM
- [ ] upgrade from previous version works
  - [ ] AWS
  - [x] Azure
  - [ ] KVM

### CODE REVIEW

File: helm/akv2k8s/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: 0.0.1
```
Comment (ubergesundheit) [inline helm/akv2k8s/Chart.yaml:2]: If the packaged app have a valid version, please update this.

Comment (T-Kukawka) [reply helm/akv2k8s/Chart.yaml:2]: so this is upstream: 
https://github.com/SparebankenVest/public-helm-charts/commit/cb0bf34a8d2563769e73fc46c2290a95ea8bad3d

i will change app version to 1.3.0

Comment (T-Kukawka) [reply helm/akv2k8s/Chart.yaml:2]: how about when i release the chart/app? should i stick to 0.0.1?

**Comment by ubergesundheit** [comment]
One heads up and one optional thing

- Heads up with the CRD. CRD yamls in the `crds` directory are applied once at first install and then are never touched again. This means if an upgrade contains an updated CRD, it won't get applied. A solution (which is being used in cert-manager-app, is a helm hook containing a Job which applies the files. [Check this link](https://github.com/giantswarm/cert-manager-app/tree/master/helm/cert-manager-app/templates/crd-install)
- Optional thing: you can generate a values schema file for the charts values using `helm schema-gen values.yaml > values.schema.json` Heads up with empty fields in the default values, you can change these to ` "type": ["null", "string"]` for example (use the correct expected type)

File: helm/akv2k8s/Chart.yaml (hunk 1 above)
Comment (ubergesundheit) [reply helm/akv2k8s/Chart.yaml]: `appVersion` is the version of the packaged app (1.3.0 in this case). You'll need to update this once you upgrade the packaged version.

`version` is the version of the chart. This is handeled by `architect` in this CI configuration

**Comment by T-Kukawka** [comment]
@ubergesundheit those are cool points that i would like to work on with next iteration of the app - i want to unblock customer - start using our app asap from the start

**Review by ubergesundheit — APPROVED** [review]
Ok if tested :)

**Comment by fiunchinho** [comment]
> * Optional thing: you can generate a values schema file for the charts values using `helm schema-gen values.yaml > values.schema.json` Heads up with empty fields in the default values, you can change these to ` "type": ["null", "string"]` for example (use the correct expected type)

is this the right plugin to use @ubergesundheit https://github.com/karuppiah7890/helm-schema-gen ? or is there a maintained version of it?

**Comment by ubergesundheit** [comment]
The repository is correct https://intranet.giantswarm.io/docs/organizational-structure/teams/halo/app-updates/helm-values-schema/

don't know about any other plugin

**Pull request merged**